libs/tree-sitter-wing/grammar.js

```diff
@@ -94,7 +95,7 @@ module.exports = grammar({
       seq(
         "struct",
         field("name", $.identifier),
-        optional(seq("extends", commaSep($.identifier))),
+        optional(seq("extends", commaSepZeroOrMore($.identifier, false))),
```

Comment:
Why not allow trailing commas here (and a couple other places)? I'm not against this change, just curious about the reasoning since we allow it in most places.

Reply:
Ah, yeah I guess I don't see a strong reason to keep the rule different in different places (also it didn't occur to me until now that the `resource A impl B, C, ... Y, Z` clause can probably extend multiple lines), so I'll revert this change